pkgs/applications/networking/cluster/minishift/default.nix

```diff
@@ -24,13 +24,19 @@ in buildGoPackage rec {
 
   nativeBuildInputs = [ pkgconfig go-bindata makeWrapper ];
   buildInputs = [ glib gtk3 libappindicator-gtk3 gpgme ostree libselinux btrfs-progs lvm2 ];
+  propagatedBuildInputs = [ openshift ];
```

Comment:
This is IMHO not needed because minishift will already have openshift in its PATH. This would include openshift in all buildInputs of packages depending on minishift.

Reply:
removed.